server/server/search/tests/test_instant_search.py

```diff
@@ -120,4 +124,52 @@ def test_normalize_string():
     assert normalize_string('rupa') == 'rupa'
     assert normalize_string('Rūpā') == 'Rupa'
     assert normalize_string('samma') == 'samma'
-    assert normalize_string('sāmma') == 'samma'
\ No newline at end of file
+    assert normalize_string('sāmma') == 'samma'
+
+
+def instant_search(query, languages):
```

Comment:
<a href="https://sourcery.ai"><img src="https://sourcery.ai/favicon.svg" height="15" alt="Sourcery logo" /></a>  **suggestion (testing):** Missing test cases for error conditions and edge cases.

Consider adding tests for invalid queries, unsupported languages, and queries that should return zero results to ensure robust error handling and edge case coverage.

```suggestion
def test_invalid_query_search():
    instant_search('', ['en', 'pli'])  # Empty query
    instant_search(' ', ['en', 'pli'])  # Query with only spaces

def test_unsupported_language_search():
    instant_search('dhamma', ['xx'])  # Unsupported language code

def test_zero_results_search():
    instant_search('nonexistentterm123', ['en', 'pli'])  # Query unlikely to return results
```